src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskProviderTest.java

```diff
@@ -101,11 +101,12 @@ private Task createTask() {
         final TaskPermissions permissions = new TaskPermissions(
             Set.of(
                 PermissionTypes.READ,
```

Comment:
Depending on the role, we get back certain permissions. With contract test it will fail if the consumer has a permission not available on the provider?

Since this is the permission, should this reflect all the permissions for this particular task?

Reply:
pact test permission added